sorc/emcsfc_snow2mdl.fd/model_grid.f

```diff
@@ -58,57 +58,53 @@ module model_grid
  real, allocatable              :: lsmask_mdl_sav (:,:)
  real                           :: resol_mdl  ! in km
 
- contains
+!> 
```

Comment:
No brief tag is needed for subroutines/functions. Remove it.
Text on first line, like this:

```
!> This is the brief comment.
!!
!! Now we elaborate with many sentences of additional documentation about this
!! function. We can go on and on.
!!
```

Reply:
Remove the brief tag (the "@brief") but leave the sentence which documents the function/subroutine.

See my example above.